go_lib/set/set.go

```diff
@@ -104,3 +105,7 @@ func (s Set) Slice() []string {
 func (s Set) Size() int {
 	return len(s)
 }
+
+func (s Set) MarshalJSON() ([]byte, error) {
+	return json.Marshal(s.Slice())
```

Comment:
you are changing old behavior of the Set marshalling. Wouldn't it be broken in other places?

Reply:
Nah, everything's fine. I checked that sets aren't used anywhere. In addition, it is strange if marshalling set to something like `{"test":{},"test-2":{}}` is expected behaviour.

I'd like to leave this be.